Q1.py

```diff
@@ -0,0 +1,16 @@
+# [“name512”, “same1example”, “joy18full”] replace the digits from string inside list
+
+lst = ['name512', 'same1example', 'joy18full']
+count = 0   
+for i in lst:
+    temp = ""
+    for j in i:
+        if j.isalpha():
+            temp += j
+        else: 
+            continue
```

Comment:
this doesn't have any addition to the iteration logic. or am I missing something?

Reply:
Hello Prathamesh, in this question what I was doing is to iterate over the each element of the list one by one and after that iterate over every character and check if that character isalpha or not, if yes, then add it in the temp variable. Please suggest me if this can be done some other way as well.